care_o_bot_robot/package.xml

```diff
@@ -13,7 +13,13 @@
   <exec_depend>cob_bringup</exec_depend>
   <exec_depend>cob_manipulation</exec_depend>
   <exec_depend>cob_navigation</exec_depend>
+  <exec_depend>htop</exec_depend>
+  <exec_depend>openssh-server</exec_depend>
+  <exec_depend>python-wstool</exec_depend>
   <exec_depend>robot_upstart</exec_depend>
+  <exec_depend>tmux</exec_depend>
+  <exec_depend>tree</exec_depend>
+  <exec_depend>vim</exec_depend>
```

Comment:
I don't like this approach too much...
Can we add a new meta-package e.g. `care_o_bot_tools` within this repo which defines the `exec_depend` to those system tools? And then we have `care_o_bot_robots` depend on `car_o_bot_tools`...